src/views/v2/tickets/item.php

```diff
@@ -117,6 +119,15 @@
 	$attributes['data-available-count'] = (string) $available_count;
 }
 
+$seat_labels = tribe( Session::class )->get_events_registrations_ticket_seat_label( $post_id, $ticket->ID );
+
+$seat_labels = implode(
```

Comment:
As mentioned above, this template should not contain any Seating feature related knowledge.